src/collection/views/Collections.tsx

```diff
@@ -1,43 +1,179 @@
-import React, { Fragment, FunctionComponent, useEffect } from 'react';
+import React, { FunctionComponent, useEffect } from 'react';
 import { connect } from 'react-redux';
 import { RouteComponentProps } from 'react-router';
-import { Dispatch } from 'redux';
+import { Link, withRouter } from 'react-router-dom';
 
+import {
+	AvatarList,
+	Button,
+	DropdownButton,
+	DropdownContent,
+	Icon,
+	MenuContent,
+	MetaData,
+	MetaDataItem,
+	Table,
+} from '@viaa/avo2-components';
 import { Avo } from '@viaa/avo2-types';
-import { Link } from 'react-router-dom';
+import { compose, Dispatch } from 'redux';
+
+import ControlledDropdown from '../../shared/components/ControlledDropdown/ControlledDropdown';
+import { formatDate } from '../../shared/helpers/formatters/date';
+import withLoading from '../../shared/hocs/withLoading';
+
 import { getCollections } from '../store/actions';
-import { selectCollections } from '../store/selectors';
+import { selectCollections, selectCollectionsLoading } from '../store/selectors';
+
+const dummyAvatars = [
+	{
+		initials: 'ES',
+		name: 'Ethan Sanders',
+		subtitle: 'Mag Bewerken',
+	},
+	{
+		initials: 'JC',
+		name: 'Jerry Cooper',
+		subtitle: 'Mag Bewerken',
+	},
+	{
+		initials: 'JD',
+		name: 'John Doe',
+		subtitle: 'Mag Bewerken',
+	},
+];
 
 interface CollectionsProps extends RouteComponentProps {
 	collections: Avo.Collection.Response[] | null;
 	getCollections: () => Dispatch;
+	loading: boolean;
 }
 
-const Collections: FunctionComponent<CollectionsProps & RouteComponentProps> = ({
+const Collections: FunctionComponent<CollectionsProps> = ({
 	collections,
 	getCollections,
+	history,
 }) => {
+	// Lifecycle
 	useEffect(() => {
 		getCollections();
 	}, [getCollections]);
 
+	// Computed
+	const mappedCollections = !!collections
+		? collections.map(c => {
+				return {
+					createdAt: formatDate(c.created_at),
+					id: c.id,
+					thumb: null,
+					title: c.title,
+					updatedAt: formatDate(c.updated_at),
+					inFolder: true,
+					access: dummyAvatars,
+					actions: true,
+				};
+		  })
+		: [];
+
+	// Render
+	const renderCell = (r: any, c: any) => {
```

Comment:
Are these row and column indexes? rowIndex and colIndex?
should have a better name than r and c

Reply:
fixed